cluster/cluster.yaml

```diff
@@ -54,7 +54,7 @@ Resources:
         Timeout: '5'
         UnhealthyThreshold: '2'
       Listeners:
-        - InstancePort: 443
+        - InstancePort: {{ if eq .ConfigItems.apiserver_proxy "true" }}8443{{ else }}443{{ end }}
```

Comment:
This will cause the API servers to be unavailable for the whole duration of the rollout because the port will be changed, but the existing nodes don't run Skipper.

Reply:
That's why I didn't guard the sidecar itself with this condition. This way going from `apiserver_proxy : false` to `apiserver_proxy: true` during a migration shouldn't have this problem, does it?

Reply:
Ah, I've missed that, sorry! Then the only potential concern is accidentally enabling this before the PR is rolled out, but we could just not do that.

Reply:
IIRC clm, will first update CF stacks and then start the rolling master nodes, so this needs 2 steps, not?